app/lib/service/openid/jwt.dart

```diff
@@ -217,16 +217,26 @@ class JwtPayload extends UnmodifiableMapView<String, dynamic> {
   /// time.
   bool verifyTimestamps([DateTime? now]) {
     now ??= clock.now();
-    if (iat != null && iat!.isAfter(now)) {
-      return false;
-    }
-    if (nbf != null && nbf!.isAfter(now)) {
-      return false;
-    }
-    if (exp != null && exp!.isBefore(now)) {
+
+    bool isNullOrAIsBeforeB(String name, DateTime? a, DateTime? b) {
+      if (a == null || b == null) {
```

Comment:
We should never allow null, please leave a comment for us to not allow that in production.

Reply:
I've updated the code to reject if any is `null`.